src/main/java/nl/beehivejava/scriptj/PluginEnvironment.java

```diff
@@ -0,0 +1,12 @@
+package nl.beehivejava.scriptj;
+
+import nl.beehivejava.scriptj.script.ScriptType;
+
+/**
+ * @author Lesley
+ */
+public interface PluginEnvironment {
+
+    ScriptType type();
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AUndocumentedApi)